authors/apps/articles/views.py

```diff
@@ -69,6 +77,8 @@ def create(self, request, *args, **kwargs):
         if Article.objects.filter(slug=slug).exists():
             slug += str(time.time()).replace('.', '')
 
+        read = self.article_read_time(body)
```

Comment:
Give this variable a better name.

Reply:
The variable name has been changed readtime